packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -285,6 +291,13 @@ export class NgOptimizedImage implements OnInit, OnChanges, OnDestroy {
    */
   @Input() srcset?: string;
 
+  /**
+   * Value of the `sizes` attribute if set on the host `<img>` element.
+   * This input is exclusively read to pass along to the preload `<link>` attribute for this image.
+   * @internal
```

Comment:
We should probably use `@nodoc` here to avoid exposing an input in the docs, but still keep it in DTS files.
```suggestion
   * @nodoc
```